packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -486,6 +488,11 @@ class Frame extends React.Component {
         this.dropdown.removeAttribute('inert');
         this.dropdown.removeAttribute('aria-hidden');
         this.selectMenu.setAttribute('tabIndex', '0');
+
+        // sets focus to select menu after select menu is positioned when opened by toggle-button click
+        if (this.selectMenu && this.shouldFocusDropdown) {
+          this.selectMenu.focus();
```

Comment:
Do we still need to call `FrameUtil.shiftFocusToMenu(this)` which "[allows time for state update to render select menu DOM before shifting focus to it](https://github.com/cerner/terra-core/blob/4e5dbb730f93dfafb61f23a539cf55186e591f8c/packages/terra-form-select/src/shared/_FrameUtil.js#L144)" instead of calling `this.selectMenu.focus()` directly?

Reply:
I don't think it is needed now. As earlier we were calling `FrameUtil.shiftFocusToMenu(this)` inside the `openDropdown`. But now we are passing focus inside the `updateDropdown` which is called by `positionDropdown` triggered by `onResize` event. So in that case the dropdown is already rendered.